assets/css/greenfield.scss

```diff
@@ -168,3 +168,127 @@ section.page-body {
     -webkit-transition: all .2s ease-in-out;
     transition: all .2s ease-in-out;
 }
+
+h3 {
+  font-family: $font-secondary-italic;
+}
+
+.beginning {
+  margin-left: 130px;
+  margin-right: 130px;
+  margin-top: 30px;
+}
+
+.beginning img {
+  margin-top: 20px;
+  height: 450px;
+}
+
+
+.approaches {
+  margin: 40px;
+  padding-top: 30px;
+  padding-bottom: 100px;
+}
+
+.approaches .item {
+  color: #FFF;
+  background-color: $brand-primary;
+  font-family: $font-primary-bold;
+  font-size: 12px;
+}
+
+.approach {
+  border-radius: 8px;
+  text-align: center;
+  font-family: $font-primary-bold;
+  padding: 20px;
+  margin-bottom: 20px;
+  margin-right: 10px;
+}
+
+.item-1 {
+  background-color: lighten($brand-primary, 17%) !important;
+}
+
+.item-2 {
+  background-color: lighten($brand-primary, 10%) !important;
+}
+
+.item-3 {
+  background-color: lighten($brand-primary, 5%) !important;
+}
+
+.working {
+  padding-top: 120px;
+  padding-bottom: 100px;
+  background-image: url(http://www.friendsoffortpointchannel.org/wp-content/uploads/2014/01/General_2-1024x600@2x.jpg);
```

Comment:
should we include this image in our assets instead of linking to an external image?